internal/test/e2e/setup.go

```diff
@@ -153,6 +159,17 @@ func (e *E2ESession) setup(regex string) error {
 	return nil
 }
 
+func (e *E2ESession) updateFSInotifyResources() error {
+	command := fmt.Sprintf("sudo sysctl fs.inotify.max_user_watches=%v && sudo sysctl fs.inotify.max_user_instances=%v", maxUserWatches, maxUserInstances)
```

Comment:
Can we call sudo via ssm?  Not sure if we are already or not...

Reply:
we did [here](https://github.com/aws/eks-anywhere/blob/main/internal/test/e2e/registryMirror.go#L37) so should be fine